app/controllers/anonymous_feedback/global_export_requests_controller.rb

```diff
@@ -11,7 +11,7 @@ def create
 
 private
   def export_request_params
-    params.permit(:from_date, :to_date).merge(notification_email: current_user.email)
+    params.permit(:from_date, :to_date).merge(notification_email: current_user.email, exclude_reviewed: !!params.permit(:exclude_reviewed))
```

Comment:
Nit: having two permits nested within each other / so close feels a bit suspect. Would this be cleaner as a helper which we then use?

```
def exclude_reviewed?
  !!params.permit(:exclude_reviewed)
end

params.permit(:from_date, :to_date).merge(notification_email: current_user.email, exclude_reviewed: exclude_reviewed?)
```